integrationtest/src/main/java/software/amazon/timestream/integrationtest/TableManager.java

```diff
@@ -35,23 +37,99 @@
+      final DeleteDatabaseRequest deleteDatabaseRequest = new DeleteDatabaseRequest();
+      deleteDatabaseRequest.setDatabaseName(database);
+      try {
+        buildWriteClient().deleteDatabase(deleteDatabaseRequest);
+      } catch (Exception exception) {
+        System.out.println(exception.getMessage());
+      }
+      buildWriteClient().createDatabase(createDatabaseRequest);
+    } catch (Exception e) {
+      System.out.println(e.getMessage());
+    }
+
+  }
+
+  /**
+   * Creates databases if not already existed.
+   * For each database, deletes the database if already existed and then creates a new one.
+   *
+   * @param databases Databases to be created
+   */
+  static void createDatabases(String[] databases) {
+    for (String database : databases) {
+      createDatabase(database);
+    }
+  }
+
+  /**
+   * Creates new tables in the database if not already existed.
+   * Deletes the table if already existed and then creates a new one.
+   *
+   * @param tables   Tables to be created
+   * @param database Database to contain the tables
+   */
+  static void createTables(String[] tables, String database) {
+    for (String table : tables) {
+      createTable(table, database);
+    }
+  }
+
+  /**
+   * Creates new tables in the databases if not already existed.
+   * Deletes the table if already existed and then creates a new one.
+   *
+   * @param tables   Tables to be created
+   * @param databases List of databases to contain the tables
+   */
+  static void createTables(String[] tables, String[] databases) {
+    for (String database : databases) {
+      createTables(tables, database);
+    }
+  }
 
   /**
-   * Creates a new table {@link Constants#TABLE_NAME} in the {@link Constants#DATABASE_NAME} if not
-   * already existed. Deletes the table if already existed and then creates a new one.
+   * Creates new tables in the database if not already existed.
+   * Deletes the table if already existed and then creates a new one.
+   *
+   * @param table    Table to be created
+   * @param database Database to contain the table
    */
-  static void createTable() {
+  static void createTable(String table, String database) {
     final CreateTableRequest createTableRequest = new CreateTableRequest();
-    createTableRequest.setDatabaseName(Constants.DATABASE_NAME);
-    createTableRequest.setTableName(Constants.TABLE_NAME);
+    createTableRequest.setDatabaseName(database);
+    createTableRequest.setTableName(table);
     final RetentionProperties retentionProperties = new RetentionProperties()
         .withMemoryStoreRetentionPeriodInHours(Constants.HT_TTL_HOURS)
         .withMagneticStoreRetentionPeriodInDays(Constants.CT_TTL_DAYS);
     createTableRequest.setRetentionProperties(retentionProperties);
     try {
       buildWriteClient().createTable(createTableRequest);
     } catch (ConflictException e) {
-      deleteTable();
+      deleteTable(Constants.TABLE_NAME, Constants.DATABASE_NAME);
```

Comment:
Why use constants value for this general function?

Reply:
fixed